src/UI/class-admin-bar.php

```diff
@@ -67,7 +67,7 @@ public function run(): void {
 	 */
 	public function admin_bar_parsely_stats_button( WP_Admin_Bar $admin_bar ): void {
 		/**
-		 * Internal Variable.
+		 * Variable.
```

Comment:
This comment adds nothing but noise now.

Reply:
It gives proper type to the variable by following PHPStan type-casting convention. If we remove this comment then type of the variable will come as `mixed` and PHPStan on `level: 9` complains about variable having mixed types.

Reply:
The `@var` below does that. The summary of `Variable` is redundant.

Reply:
Removing description and just having `@var` in doc comment gives following error from PHPCS

```
Missing short description in doc comment
```